.tekton/scripts/check-partner-tasks.sh

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env bash
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd "$SCRIPTDIR/../.." || exit 1
+
```

Comment:
Will it be good to consider the fact that partner users can also run the script locally before sending the PR , it will reduce the number of pipeline runs (if partner users try trail and error method to fix the issues in the task), one approach I think maybe by passing the task directory as an argument to the script. WDYT?

Reply:
User is able to run `.tekton/scripts/check-partner-tasks.sh` directly. I'd like to keep the simplicity for now.

Reply:
Do you mean without modifying the script?

Reply:
Yes, for now, not add an argument to allow specifying a directory to check.

Reply:
okay got it. Thanks.